sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -510,6 +510,10 @@ def from_estimator(
         --------
         partial_dependence : Compute Partial Dependence values.
 
+        Notes
+        -----
+        NaNs values in the `X` matrix are ignored when computing the partial dependence.
```

Comment:
```suggestion
    NaN values in `X` are ignored when computing the partial dependence.
```
I don't think we need the `s` on NaN. IMHO "NaN values" is "one word".

What do you think about replacing the "the X matrix" with just "X"?